lib/msf/ui/console/command_dispatcher/db/analyze.rb

```diff
@@ -44,23 +47,23 @@ def cmd_analyze(*args)
       host_ids.each do |id|
```

Comment:
Not a blocker for this PR, but is it worth improving the logging for the scenario where there's hosts available but nothing is suggested?

```
msf6 auxiliary(scanner/smb/smb_version) > hosts

Hosts
=====

address          mac                name            os_name       os_flavor  os_sp  purpose  info  comments
-------          ---                ----            -------       ---------  -----  -------  ----  --------
192.168.18.118   00:00:00:00:00:00  VAGRANT-2008R2
192.168.222.151  00:00:00:00:00:00                  Windows 2008                    server

msf6 auxiliary(scanner/smb/smb_version) > analyze
msf6 auxiliary(scanner/smb/smb_version) > analyze
msf6 auxiliary(scanner/smb/smb_version) > 
```

I initially thought the command was breaking silently when it gave no output 😄

Reply:
Yeah, that's existing behaviour, but it seems worthwhile to fix. Probably worth having separate messages for no hosts, hosts with no vulns, and hosts with vulns that don't match anything.